src/types/plugin-inputs.ts

```diff
@@ -12,18 +13,41 @@ export interface PluginInputs<T extends WebhookEventName = SupportedEvents> {
   ref: string;
 }
 
+function thresholdType(options?: StringOptions) {
+  return T.Transform(
+    T.String({
+      // Matches a pattern like [decimal] [unit], e.g. 3.25 hours
+      pattern: /^\s*\d+(\.\d+)?\s+\S+\s*$/.source,
```

Comment:
```suggestion
      pattern: /^\s*\d+(\.\d+)?\s*\S+\s*$/.source,
```
`ms` also supports strings like `1h`. anyway I wouldn't even test for a pattern because `ms` handles everything